cogstat/cogstat_hyp_test.py

```diff
@@ -273,8 +274,63 @@ def wilcox_sign_test(pdf, data_measlevs, var_name, value=0):
 
 ### Variable pair ###
 
+def multivariate_normality(pdf, var_names, group_name='', group_value='', alt_data=None):
+    """Henze-Zirkler test of multivariate normality.
+
+        Parameters
+        ----------
+        pdf : pandas dataframe
+            It is sufficient to include only the relevant variables. It is assumed that nans are dropped.
+        var_names : str
+            Name of the variables to test.
+        group_name : str
+            Name of grouping variable if part of var_name should be checked. Otherwise ''.
+        group_value : str
+            Name of the group in group_name, if grouping is used.
+        alt_data : dataframe
+            If alt_data is specified, this one is used instead of self.data_frame.
+            This could be useful if some other data should be dropped, e.g., in variable comparison,
+            where cases are dropped based on missing cases in other variables.
+
+        Returns
+        -------
+        bool
+            True if normality is true.
+        html text
+            Output in APA format.
+
+        """
+
+    text_result = ''
+    if repr(alt_data) == 'None':
+        # bool(pd.data_frame) would stop on pandas 0.11
+        # that's why this weird alt_data check
+        temp_data = pdf
+    else:
+        temp_data = alt_data
+
+    if group_name:
+        data = temp_data[temp_data[group_name] == group_value][var_names].dropna()
+    else:
+        data = temp_data[var_names].dropna()
+    if len(set(data)) == 1:
+        return False, _('Normality cannot be checked for constant variable in %s%s.\n' %
+                        (var_names, ' (%s: %s)' % (group_name, group_value) if group_name else ''))
```

Comment:
```suggestion
        return False, _('Normality cannot be checked for constant variable in %s%s.\n') %
                        (var_names, ' (%s: %s)' % (group_name, group_value) if group_name else '')
```

Reply:
When the test cannot be run, shouldn't we return None instead of False? If so, please, fix this in all cases, and extend the string doc too.